### PR TITLE
Add google/wire to the list of DI libraries

### PR DESCRIPTION
**Pull request by azazi-sa**
> Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards) before sending pull request. Thanks!

**Please provide package links to:**

- repo link (github.com, gitlab.com, etc): https://github.com/google/wire
- pkg.go.dev: https://pkg.go.dev/github.com/google/wire
- goreportcard.com: https://goreportcard.com/report/github.com/google/wire
- coverage service link ([codecov](https://codecov.io/), [coveralls](https://coveralls.io/), [gocover](http://gocover.io/) etc.): https://app.codecov.io/gh/google/wire

**Note**: _that new categories can be added only when there are 3 packages or more._

**Make sure that you've checked the boxes below before you submit PR:**
_not every repository (project) will fit into every option, but most projects should_

- [x] I have added my package in alphabetical order.
- [x] I have an appropriate description with correct grammar.
- [x] I know that this package was not listed before.
- [x] I have added pkg.go.dev link to the repo and to my pull request.
- [x] I have added coverage service link to the repo and to my pull request.
- [x] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/main/CONTRIBUTING.md#quality-standards).

Thanks for your PR, you're awesome! :+1:

### CODE REVIEW

**Comment by avelino** [comment]
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body links content check:
- godoc.org or pkg.go.dev: True
- goreportcard.com: False
- coverage: True

> your project is about to be approved, it's under revision, _it may take a few days_

File: README.md [hunk 1]
```diff
@@ -1420,12 +1420,14 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
...
 * [gocontainer](https://github.com/vardius/gocontainer) - Simple Dependency Injection Container.
 * [goioc/di](https://github.com/goioc/di) - Spring-inspired Dependency Injection Container.
+* [google/wire](https://github.com/google/wire) - Automated Initialization in Go.
 * [HnH/di](https://github.com/HnH/di) - DI container library that is focused on clean API and flexibility.
 * [kinit](https://github.com/go-kata/kinit) - Customizable dependency injection container with the global mode, cascade initialization and panic-safe finalization.
 * [linker](https://github.com/logrange/linker) - A reflection based dependency injection and inversion of control library with components lifecycle support.
 * [nject](https://github.com/muir/nject) - A type safe, reflective framework for libraries, tests, and http endpoints, and service startup.
 * [wire](https://github.com/Fs02/wire) - Strict Runtime Dependency Injection for Golang.
 
+
```
Comment (phanirithvij) [inline README.md:1430]:
```suggestion

```

**Comment by adebobo19** [comment]
Google and wire

**Comment by adebobo19** [comment]
Send me Bitcoin now

**Pull request merged**